WFInfo/Login.xaml.cs

```diff
@@ -1,124 +1,159 @@
+            {
+                try
+                {
+                    await Main.dataBase.GetUserLogin(Email.Text, Password.Password);
+                }
+                catch (Exception ex)
+                {
+                    Main.dataBase.JWT = null;
+                    Settings.Save();
+                    Main.AddLog("Couldn't login: " + ex);
+                    var StatusMessage = "Status"; //StatusMessage = text to display on StatusUpdate() AND the error box under login 
+                    var StatusSeverity = 0; //StatusSeverity = Severity for StatusUpdate()
+                    if (ex.Message.Contains("email"))
+                    {
+                        if (ex.Message.Contains("app.form.invalid"))
+                        {
+                            StatusMessage = "Invalid email form";
+                            StatusSeverity = 2;
+
+                        }
+                        else
+                            StatusMessage = "Unknown email";
+                            StatusSeverity = 1;
+                    }
+                    else if (ex.Message.Contains("password"))
+                    {
+                        StatusMessage = "Wrong password";
+                        StatusSeverity = 1;
+                    }
+                    else if (ex.Message.Contains("could not understand"))
+                    {
+                        StatusMessage = "Severe issue, server did not understand request";
+                        StatusSeverity = 1;
+                    }
+                    else
+                    {
+                        StatusMessage = "Too many requests";
+                        StatusSeverity = 1; //default to too many requests
+                    }
+                    Main.SignOut();
+                    Main.StatusUpdate(StatusMessage, StatusSeverity); //Changing WFinfo status
+
+                    switch (StatusSeverity)
+                    { // copy/paste from Main.cs (statusChange())
+                        case 0: //default, no problem
+                            Error.Foreground = new SolidColorBrush(Color.FromRgb(177, 208, 217));
+                            break;
+                        case 1: //severe, red text
+                            Error.Foreground = Brushes.Red;
+                            break;
+                        case 2: //warning, orange text
+                            Error.Foreground = Brushes.Orange;
+                            break;
+                        default: //Uncaught, big problem
+                            Error.Foreground = Brushes.Yellow;
+                            break;
+                    }
+                    Error.Text = StatusMessage; //Displaying the error under the text fields
+                    if(Error.Visibility != Visibility.Visible)
+                    {
+                        Height += 20;
+                        BackupButton.VerticalAlignment = VerticalAlignment.Center;

```

Comment:
No need to change the vertical alignment if you align the item at the bottom. It'll dynamically stay at the bottom

Reply:
i tried it, it seems like it didnt, but i may used some wrong functions